versions/3.0.md

```diff
@@ -8,15 +8,15 @@ This document is licensed under [The Apache License, Version 2.0](http://www.apa
 
 ## Introduction
 
-The OpenAPI Specification (OAS) defines a standard, language-agnostic interface to RESTful APIs which allows both humans and computers to discover and understand the capabilities of the service without access to source code, documentation, or through network traffic inspection. When properly defined, a consumer can understand and interact with the remote service with a minimal amount of implementation logic.
+The OpenAPI Specification (OAS) defines a standard, language-agnostic interface to RESTful APIs which allows both humans and computers to discover and understand the capabilities of the service without access to source code, documentation, or through network traffic inspection. When properly documented, a consumer can understand and interact with the remote service with a minimal amount of implementation logic.
 
-An OpenAPI definition can then be used by documentation generation tools to display the API, code generation tools to generate servers and clients in various programming languages, testing tools and many other use cases.
+An OpenAPI document can then be used by documentation generation tools to display the API, code generation tools to generate servers and clients in various programming languages, testing tools and many other use cases.
```

Comment:
I still feel that 'define' is the best term between the three. "document" is valid but (like description) has other meanings/associations which I think are distracting. In particular, phrases like "When properly documented" feel too much like "When properly described" which to me infers non-nominal prose description rather than an a source definition.  This sentence itself highlights the problem: "An OpenAPI document can then be used by documentation generation tools to display the API" -- i.e. *use the document to generate the documentation*, when *use the **definition** to generate the documentation* feels more correct. (I do see how a UI or web page rendering is just another *representation* of the OpenAPI document.)

"document" is better that "description" but "definition" will yield the least amount of confusion.